server/src/api_endpoints/personas/permissions.js

```diff
@@ -13,6 +13,20 @@ const getPersonaPermissions = async (req, res, next) => {
   }
 }
 
+const createPersonaPermissions = async (req, res, next) => {
+  try {
+    const { persona_key: personaKey } = req.params
+    await checkPersonaExists(personaKey)
+    await dbAdminPersona.insertPersonaPermissions(personaKey, req.body)
+    const locationUrl = `${req.protocol}://${req.get('host')}${req.originalUrl}`
```

Comment:
@k1ch how come you are returning the Location header? i agree that 201 is the right status code. 

Is this something that we'll start to do for other "create" APIs?

Reply:
So this endpoint is creating records on `personapermissions` table. However we are interested in the actual permissions, so instead of returning them and making additional query to DB, we set the Locations which can be used to to retrieve all the permissions assigned to the persona as proposed [here](https://github.com/DMGT-TECH/the-usher-server/issues/60#issuecomment-1777921085). (we return `201` instead of `204` to follow [the official documents](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Location))